awx_collection/tests/integration/targets/tower_project_update/tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+---
+- name: Generate a random string for test
+  set_fact:
+    test_id: "{{ lookup('password', '/dev/null chars=ascii_letters length=16') }}"
+  when: test_id is not defined
+
+- name: Generate names
+  set_fact:
+    project_name1: "AWX-Collection-tests-tower_project_update-project-{{ test_id }}"
+
+- name: Create a git project without credentials without waiting
+  tower_project:
+    name: "{{ project_name1 }}"
+    organization: Default
+    scm_type: git
+    scm_url: https://github.com/ansible/test-playbooks
+    wait: false
+  register: result
+
+- assert:
+    that:
+      - result is changed
+
+- name: Update a project without waiting
+  tower_project_update:
+    name: "{{ project_name1 }}"
+    organization: Default
+    wait: false
+  register: result
+
+- assert:
+    that:
+      - result is changed
+
+- name: Update a project and wait
+  tower_project_update:
+    name: "{{ project_name1 }}"
+    organization: Default
+    wait: true
+  register: result
+
+- assert:
+    that:
+      - result is changed
```

Comment:
If this returns the job data we should assert that it's done (to verify that `wait` actually works), im not exactly sure what data we get back here

Reply:
I've updated this to successful here was the result info
```
{
    "result": {
        "changed": true,
        "failed": false,
        "id": 925,
        "status": "successful",
    }
}
```